.github/workflows/buildExamples.yml

```diff
@@ -39,3 +39,28 @@ jobs:
 
       - name: Build examples
         run: yarn workspaces foreach -p run build-storybook
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
+
+      - uses: actions/cache@v2
```

Comment:
Is there an advantage to this vs. using `cache: yarn` on `setup-node` (https://github.com/actions/setup-node#caching-packages-dependencies)?

Reply:
Nope, I had a version where I swapped this for cache: yarn but I reverted it since the change wasn’t strictly necessary. Happy to change back.